assignments/pr_review_tutorial/costa/README.md

```diff
@@ -1,6 +1,5 @@
-Ok so I guess you are reading this cuz you wanna use my code. There are some
-functions that do stuf and thats:
-
+You will find in the file simple_functions.py that it contains two functions.
+The first function, fibonacci, takes an integer as its input parameter, max. This input is the maximum value of a fibonacci sequence. Next, we have an array of integers called "values". This array starts with just two values: 0 and 1. The subsequent while loop takes the sum of the last two integers in the array. 
```

Comment:
following Brigitta's lead, time for tick marks around names like this:  triple ticks <words> triple ticks
```simple_functions.py```